content/blog/2022/06/2022-6-15-svg-icon-migration.adoc

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "SVG icon migration (breaking changes!)"
+tags:
+- svg-migration
+- community
+- ux
+author: kmartens27
+description: >
+  Starting with Jenkins 2.333, Jenkins core has migrated from `GIF` and `PNG` icons to `SVG` icons.
+---
+
+== Summary
+
+Beginning with Jenkins v2.333 (released on June 15, 2022), Jenkins core has upgraded from `GIF` and `PNG` icons to `SVG` icons.
+Plugins that will be affected by this change are tracked in a link:https://docs.google.com/spreadsheets/d/1PxlgT11_uDyTzPch8zWn3PDxLUIAab21ILmJ17zCzBk/edit#gid=1308179162[spreadsheet] found in link:https://issues.jenkins.io/browse/JENKINS-68251[JENKINS-68251].
+Plugins that do not interact with icons are unaffected by this change.
+
+== Motivation

```

Comment:
I think you can call this whatever you want, but I would consider calling this "Rationale" or "Explanation". Motivation is just fine though. Consider this food for thought.

Reply:
```suggestion
== Background

```

Reply:
Thanks @cari-mason that makes sense, I've changed it to Background to make a bit more sense!